docs/fundamentals/monorepo.md

```diff
@@ -0,0 +1,296 @@
+# Building like Google: Monorepos and Automation
+
+In this guide you will look at one of the most interesting parts of Nx. The part that can make so many things so much easier, that it can have a transformative effect on a team and even on an organization.
+
+What is it?
+
+It is a "monorepo" way of building applications.
+
+Working with multiple applications and libraries is difficult. From Google to Facebook, Uber, Twitter and more, a good amount of large software companies handle this challenge by taking a monorepo approach. And they have been doing so for years. These are some of the advantages this approach provides:
+
+- Everything at that current commit works together
+- Easy to split code into lib modules, easy to compose them
+- Easier dependency management
+- One build setup (Angular CLI)
```

Comment:
I feel like removing `(Angular CLI)` makes this more of a fundamental article separate from Nx

```suggestion
- One toolchain setup
```